src/main/java/org/apache/commons/lang3/reflect/MethodUtils.java

```diff
@@ -327,6 +327,7 @@ public static Object invokeExactMethod(final Object object, final String methodN
     public static Object invokeExactMethod(final Object object, final String methodName, Object[] args, Class<?>[] parameterTypes)
         throws NoSuchMethodException, IllegalAccessException, InvocationTargetException {
         Objects.requireNonNull(object, "object");
+        Objects.requireNonNull(object, "methodName");
```

Comment:
The null check should be on `methodName`

Reply:
-1 You lose the better exception message below, specifically, you lose the class name. Follow the code path and you'll see that `getAccessibleMethod()` returns null if the `methodName` is null.